src/python/model/README.md

```diff
@@ -0,0 +1,68 @@
+# Qodana imitation model 
```

Comment:
Please, transfer your model into `qodana` directory and call it something like this: `initation_model`